src/Setup.elm

```diff
@@ -169,23 +135,21 @@ view model =
         startYear =
             smallerYear model.slider1 model.slider2
     in
-    Element.layout [] <|
-        Element.column [ Element.centerX, Element.moveDown 200, Element.scale 1.9 ]
-            [ yearsInput
-                model.slider1
-                (Input.labelAbove [ Font.color <| rgb255 0 0 200, Font.size 43, Element.centerX ] (text "Years:"))
-                (\number -> Slider1 <| round number)
-            , yearsInput
-                model.slider2
-                (Input.labelBelow [ Font.color <| rgb255 0 0 200, Font.size 40, Element.alignLeft ]
-                    (text <| "From " ++ String.fromInt startYear ++ " To " ++ String.fromInt endYear)
-                )
-                (\number -> Slider2 <| round number)
-            , Element.column [ Element.centerX, Element.scale 1.1, Element.moveDown 25 ]
-                [ factorInput model.districtFactor "District Factor:       " (\number -> DistrictFactor <| stringToFloat number)
-                , factorInput model.offSeasonFactor "Off Season Factor:" (\number -> OffSeasonFactor <| stringToFloat number)
-                , factorInput model.playOffFactor "PlayOff Factor:      " (\number -> PlayOffFactor <| stringToFloat number)
-                , factorInput model.einsteinFactor "Einstein Factor:     " (\number -> EinsteinFactor <| stringToFloat number)
-                , continueButton
-                ]
+    Element.column []
+        [ yearsInput
+            model.slider1
+            (Input.labelAbove [ Font.color <| rgb255 0 0 200, Font.size 43, Element.centerX ] (text "Years:"))
+            (\number -> Slider1 <| round number)
+        , yearsInput
+            model.slider2
+            (Input.labelBelow [ Font.color <| rgb255 0 0 200, Font.size 40, Element.alignLeft ]
+                (text <| "From " ++ String.fromInt startYear ++ " To " ++ String.fromInt endYear)
+            )
+            (\number -> Slider2 <| round number)
+        , Element.column [ Element.centerY, Element.centerX, Element.scale 1.1, Element.moveDown 25 ]
+            [ factorInput model.districtFactor "District Factor:       " (\number -> DistrictFactor <| stringToFloat number)
+            , factorInput model.offSeasonFactor "Off Season Factor:" (\number -> OffSeasonFactor <| stringToFloat number)
+            , factorInput model.playOffFactor "PlayOff Factor:      " (\number -> PlayOffFactor <| stringToFloat number)
+            , factorInput model.einsteinFactor "Einstein Factor:     " (\number -> EinsteinFactor <| stringToFloat number)
```

Comment:
Don't use whitespace for layout. Use `width`, `padding` and `spacing` properly instead.